src/launch_common.py

```diff
@@ -83,11 +82,18 @@ def add_argument_release_override(parser):
                         default=0, type=int, action=ReleaseOverrideAction)
 
 
+class PaymentOffsetAction(argparse.Action):
+    def __call__(self, parser, namespace, values, option_string=None):
+        if not (values >= 0 and values < 4096):
```

Comment:
4096 should be a global constant as it is the number of blocks per cycle. No magic numbers